### ISSUE TITLE
Can't run tests

### ISSUE THREAD

**Issue by rob-deutsch** [adbancroft/TunerStudioIniParser#1]
Hi there! I'm trying to get this code to work on my machine, but with little success.

**Reproduction steps:**

```
git clone https://github.com/adbancroft/TunerStudioIniParser.git
cd TunerStudioIniParser
pip install more-itertools lark
python -m unittest tests/test_all_files.py
```

**Expected results:**
Running of unit tests

**Actual results:**
```
(tsini) % python -m unittest tests/test_all_files.py                                        
Loading grammar from cache: /Users/rob/tmp/tsini/TunerStudioIniParser/TsIniParser/pre_processor.lark.cache
Loading grammar from cache: /Users/rob/tmp/tsini/TunerStudioIniParser/TsIniParser/ts_ini.lark.cache
Parsing /Users/rob/tmp/tsini/TunerStudioIniParser/tests/Test_Files/MS2ExtraSerial324.ini
ELoading grammar from cache: /Users/rob/tmp/tsini/TunerStudioIniParser/TsIniParser/pre_processor.lark.cache
Loading grammar from cache: /Users/rob/tmp/tsini/TunerStudioIniParser/TsIniParser/ts_ini.lark.cache
.
======================================================================
ERROR: test_allfiles (tests.test_all_files.test_all_files)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/rob/tmp/tsini/TunerStudioIniParser/tests/test_all_files.py", line 42, in test_allfiles
    parse_file(ini_file, parser)
  File "/Users/rob/tmp/tsini/TunerStudioIniParser/tests/test_all_files.py", line 13, in parse_file
    ini = file.read()
  File "/usr/local/Cellar/python@3.9/3.9.6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb0 in position 12385: invalid start byte

----------------------------------------------------------------------
Ran 2 tests in 1.570s

FAILED (errors=1)
```

**Comment by adbancroft** [comment]
Reproduced: looks like some of the test files have a non-utf8 encoding

**Comment by adbancroft** [comment]
Fixed with d2a7ab3a408b8670bc1affd5d20c15c0c5ea2463

**Comment by rob-deutsch** [comment]
Thanks!
